environment/environment.go

```diff
@@ -13,7 +13,7 @@ func LoadEnv(relPath string) {
 	env := os.Getenv("MATTERHORN_ENV")
 	if "" == env {
 		// Default to "local"
-		env = "dev"
+		env = "local"
```

Comment:
I feel like defaulting to non-prod if the env is not defined is a bit dangerous considering that removes the need to authenticate for a bunch of different request types. Consider throwing an exception if env is not defined.

Reply:
Good catch, this doesn't account for the fact that there are values being set as environment variables. Will change this.